docker/helm/values.yaml

```diff
@@ -36,35 +36,36 @@ workerReplicas: 5
 # celeborn release version
 celebornVersion: 0.1.1
 
+gid: 10006
+
 # celeborn configurations
 celeborn:
-  # please update celeborn.worker.storage.dirs to disk mount path on k8s node
+  celeborn.ha.master.ratis.raft.server.storage.dir: /mnt/rss_ratis/
   celeborn.worker.storage.dirs: /mnt/disk1,/mnt/disk2,/mnt/disk3,/mnt/disk4
   celeborn.push.replicate.enabled: true
+  celeborn.metrics.enabled: true
   celeborn.master.metrics.prometheus.port: 9098
+  celeborn.worker.metrics.prometheus.port: 9096
   celeborn.worker.monitor.disk.enabled: false
-  rss.rpc.io.serverThreads: 64
-  rss.worker.fetch.chunk.size: 8m
-  rss.rpc.io.numConnectionsPerPeer: 2
+  celeborn.worker.fetch.chunk.size: 8m
+  celeborn.rpc.io.serverThreads: 64
+  celeborn.rpc.io.numConnectionsPerPeer: 2
+  celeborn.rpc.io.clientThreads: 64
+  celeborn.rpc.dispatcher.numThreads: 4
   celeborn.worker.flush.buffer.size: 256K
-  celeborn.metrics.enabled: true
-  rss.push.io.threads: 32
   celeborn.worker.fetch.io.threads: 32
+  celeborn.push.io.threads: 32
```

Comment:
we should add "celeborn.worker.fetch.io.threads: 32" here because the PVC will be regarded as one disk.